src/sfctl/tests/chaos_test.py

```diff
@@ -0,0 +1,66 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Custom Chaos command related tests"""
+import unittest
+import sfctl.custom_chaos as sf_c
+
+#pylint: disable=invalid-name
```

Comment:
Why do you need to disable this?

Reply:
Copy-paste from health_test.py, removed now.